Tests/IndexStoreDBTests/IndexTests.swift

```diff
@@ -227,6 +227,55 @@ final class IndexTests: XCTestCase {
   #endif
   }
 
+  /// Same as `testHermeticExplicitOutputUnits` but with a remapping to make output paths relative.
```

Comment:
Unfortunate we have to add a whole new test just to change mappings... suppose it makes sense though.